client/src/pages/Messages.js

```diff
@@ -0,0 +1,347 @@
+      },
+      body: "Lorem Ipsum!",
+      seen: true,
+    },
+    {
+      sentBy: {
+        _id: "user2",
+        profilePictureUrl:
+          "https://cdn.iconscout.com/icon/free/png-256/chef-2309868-1943778.png",
+      },
+      body: "Lorem Ipsum!",
+      seen: true,
+    },
+    {
+      sentBy: {
+        _id: "user1",
+        profilePictureUrl:
+          "https://i1.sndcdn.com/avatars-000624434067-r5kehg-t500x500.jpg",
+      },
+      body: "Do you want to book appointment?",
+      seen: true,
+    },
+    {
+      sentBy: {
+        _id: "user1",
+        profilePictureUrl:
+          "https://i1.sndcdn.com/avatars-000624434067-r5kehg-t500x500.jpg",
+      },
+      body: "Do you want to book appointment?",
+      seen: true,
+    },
+    {
+      sentBy: {
+        _id: "user1",
+        profilePictureUrl:
+          "https://i1.sndcdn.com/avatars-000624434067-r5kehg-t500x500.jpg",
+      },
+      body: "Do you want to book appointment?",
+      seen: true,
+    },
+    {
+      sentBy: {
+        _id: "user1",
+        profilePictureUrl:
+          "https://i1.sndcdn.com/avatars-000624434067-r5kehg-t500x500.jpg",
+      },
+      body: "Do you want to book appointment?",
+      seen: true,
+    },
+    {
+      sentBy: {
+        _id: "user1",
+        profilePictureUrl:
+          "https://i1.sndcdn.com/avatars-000624434067-r5kehg-t500x500.jpg",
+      },
+      body: "Do you want to book appointment?",
+      seen: true,
+    },
+    {
+      sentBy: {
+        _id: "user1",
+        profilePictureUrl:
+          "https://i1.sndcdn.com/avatars-000624434067-r5kehg-t500x500.jpg",
+      },
+      body: "Do you want to book appointment?",
+      seen: true,
+    },
+    {
+      sentBy: {
+        _id: "user1",
+        profilePictureUrl:
+          "https://i1.sndcdn.com/avatars-000624434067-r5kehg-t500x500.jpg",
+      },
+      body: "Do you want to book appointment?",
+      seen: true,
+    },
+    {
+      sentBy: {
+        _id: "user1",
+        profilePictureUrl:
+          "https://i1.sndcdn.com/avatars-000624434067-r5kehg-t500x500.jpg",
+      },
+      body: "Are you there?",
+      seen: true,
+    },
+  ];
+
+  const [activeMessageItem, setActiveMessageItem] = useState({});
+
+  // this will set the active message state
+  // will be used those state to chat area
+  const handleSelectedMessageItem = (message) => {
+    setActiveMessageItem(message);
+  };
+
+  return (
+    <Grid container>
+      {/* SIDEBAR */}
+      <Grid item md={2} sm={4} xs={12}>
+        <Box height="100vh">
+          {usersChat.chats.map((chat, i) => {
+            chat["i"] = i;
+            return (
+              <MessageItems
+                chat={chat}
+                name={chat.chattingWith.user.name}
+                isOnline={true}
+                lastMessage={chat.lastMessage.body}
+                profilePictureUrl={chat.chattingWith.user.profilePictureUrl}
+                key={i}
```

Comment:
It's preferable to use a [unique identifier](https://reactjs.org/docs/lists-and-keys.html#keys) rather than index as the key.
You can build one with the index such as `${name}chat${i}`

Reply:
updated. thank you.